extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/contacts/GoogleContactsExporter.java

```diff
@@ -116,7 +116,7 @@ private static void convertToVCardNamesAndPopulate(VCard vCard, List<Name> perso
     for (Name personName : personNames) {
       StructuredName structuredName = convertToVCardNameSingle(personName);
       Boolean isNamePrimary = personName.getMetadata().getPrimary();
-      if (isNamePrimary != null && isNamePrimary) {
+      if (primaryStructuredName == null && isNamePrimary != null && isNamePrimary) {
```

Comment:
This overall is an OK change, but I'm wondering if theres some other structure we can use to capture this? The main difference that's happening here iiuc, is whether we use the first or the the last primary name we see. 

@holachuy  for input

Reply:
Seems strange to have more than one primary name. I don't oppose either case.